docs/releases/v13/upgrade-v13.0.md

```diff
@@ -26,22 +26,6 @@ invenio-cli, version 1.8.0
     In case you are not inside a virtual environment, make sure that you prefix each `invenio`
     command with `pipenv run`.
 
-**Local development**
-
-Changing the Python version in your development environment highly
-depends on your setup, so we won't cover it here.
-One way would be to use [PyEnv](https://github.com/pyenv/pyenv).
-
-!!! warning "Risk of losing data"
-
-    Your virtual environment folder a.k.a., `venv` folder, may contain uploaded files. If you kept the default
-    location, it is in `<venv folder>/var/instance/data`. If you need to keep those files,
-    make sure you copy them over to the new `venv` folder in the same location.
-    The command `invenio files location list` shows the file upload location.
-
-If you upgraded your python version, you should recreate your virtual environment before
-running `invenio-cli` or `pipenv` commands below.
-
```

Comment:
I removed this section since it seemed to be about updating the Python version which I would place as out-of-scope for an InvenioRDM upgrade (especially this one since v13 supports the same versions of Python as v12).

Reply:
about upgrading the python version, yes we can remove this, but we should maybe say something about the python3.9 end of life during the v13 lifetime.
and we should not remove the section about explaining that removing the `<venv folder>/var/instance/data` can remove the data